src/main/java/graphql/util/TraverserVisitorStub.java

```diff
@@ -0,0 +1,19 @@
+package graphql.util;
+
+import graphql.Internal;
+
+@Internal
+public class TraverserVisitorStub<T> implements TraverserVisitor<T> {
+
+
+    @Override
+    public TraversalControl enter(TraverserContext<T> context) {
+        return TraversalControl.CONTINUE;
+    }
+
+    @Override
+    public TraversalControl leave(TraverserContext<T> context) {
+        return TraversalControl.CONTINUE;
+    }
+
+}
```

Comment:
Sometimes I wont if default methods on interfaces is not a better way to do this.  eg some vistor extends TraverserVisitor but its all implemented like the above.  So you only need 1 interface type and not the interface and stub

Reply:
yea ... don't have an answer to that, maybe it is better do have just default methods.